src/app/src/components/annotations/utils/annotatorsettings.tsx

```diff
@@ -41,11 +48,29 @@ export default class AnnotatorSettings extends Component<
     };
 
     /* Initialize filters */
-    this.imageElement = this.props.image;
-    this.imageElement.style.filter =
+    this.getImageElement().style.filter =
       "brightness(100%) contrast(100%) saturate(100%)";
   }
 
+  /**
+   * Function to get the current Video or Image element
+   * @returns HTML Element that targets the Current Video/ Image
+   */
+  private getImageElement = (): any => {
+    if (
+      document.querySelector(
+        ".leaflet-pane.leaflet-overlay-pane img.leaflet-image-layer"
+      ) !== null
+    ) {
+      return document.querySelector(
+        ".leaflet-pane.leaflet-overlay-pane img.leaflet-image-layer"
+      );
+    }
+    return document.querySelector(
+      ".leaflet-pane.leaflet-overlay-pane video.leaflet-image-layer"
+    );
+  };
+
```

Comment:
@virtuoso98 I understand the intuition, but you might be more clear doing ternary if you are sure that the element on the screen is binary.

```
private getImageElement = (): any => document.querySelector(
        ".leaflet-pane.leaflet-overlay-pane img.leaflet-image-layer"
      ) ?? document.querySelector(
      ".leaflet-pane.leaflet-overlay-pane video.leaflet-image-layer"
    );

Reply:
Using JavaScript Nullish Coalescing Operator - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
Changes done. Thanks for the shorthand!